.drone.yml

```diff
@@ -4,7 +4,8 @@ workspace:
 
 services:
   web:
-    image: fpfis/httpd-php-dev:7.1
+    # @todo: move back to 7.1 after Docker image is fixed upstream.
```

Comment:
What is the reason for the change?

Reply:
This has been added and reverted later, so, discard this please.